src/watchmaker/workers/salt.py

```diff
@@ -151,10 +151,10 @@ def _prepare_for_install(self):
             self.salt_working_dir_prefix
         )
 
-        if (
-            self.salt_debug_log and
-            self.salt_debug_log != 'None'
-        ):
+        self.salt_debug_log = watchmaker.utils.config_none_deprecate(
```

Comment:
How about moving these to the `__init__` where we set `self.salt_debug_log` in the first place?

Reply:
Absolutely. Makes more sense.